app/features/accounts/account-selector.tsx

```diff
@@ -8,12 +8,14 @@ import {
   DrawerTrigger,
 } from '~/components/ui/drawer';
 import { Money } from '~/lib/money';
-import type { Account } from './account';
+import type { Account, CashuAccount } from './account';
 import { AccountTypeIcon } from './account-icons';
 
-type AccountWithBadges<T extends Account = Account> = T & {
+export type AccountWithBadges<T extends Account = Account> = T & {
+  /** Text to display as a badge in the account selector */
   badges?: string[];
 };
+export type CashuAccountWithBadges = AccountWithBadges<CashuAccount>;
```

Comment:
this type doesn't seem like it belongs there. can we move it to the hook file where it is used?